aea/helpers/profiling.py

```diff
@@ -46,22 +46,27 @@
 _default_logger = logging.getLogger(__file__)
 
 if platform.system() == "Windows":  # pragma: nocover
-    import win32process  # type: ignore  # pylint: disable=import-error
 
     WIN32_PROCESS_TIMES_TICKS_PER_SECOND = 1e7
 
     def get_current_process_memory_usage() -> float:
         """Get current process memory usage in MB."""
+        import win32process  # type: ignore  # pylint: disable=import-error,import-outside-toplevel
```

Comment:
Suspect also need to import `win32timezone` - based on logs

Reply:
Fixed here https://github.com/valory-xyz/open-aea/pull/298/commits/08a34f52efc732f3e39309562f67863e8cf711c0